lm_eval/tasks/arabic_leaderboard_complete/README.md

```diff
@@ -0,0 +1,254 @@
+      * `arabic_mmlu_abstract_algebra`
+      * `arabic_mmlu_anatomy`
+      * `arabic_mmlu_astronomy`
+      * `arabic_mmlu_business_ethics`
+      * `arabic_mmlu_clinical_knowledge`
+      * `arabic_mmlu_college_biology`
+      * `arabic_mmlu_college_chemistry`
+      * `arabic_mmlu_college_computer_science`
+      * `arabic_mmlu_college_mathematics`
+      * `arabic_mmlu_college_medicine`
+      * `arabic_mmlu_college_physics`
+      * `arabic_mmlu_computer_security`
+      * `arabic_mmlu_conceptual_physics`
+      * `arabic_mmlu_econometrics`
+      * `arabic_mmlu_electrical_engineering`
+      * `arabic_mmlu_elementary_mathematics`
+      * `arabic_mmlu_formal_logic`
+      * `arabic_mmlu_global_facts`
+      * `arabic_mmlu_high_school_biology`
+      * `arabic_mmlu_high_school_chemistry`
+      * `arabic_mmlu_high_school_computer_science`
+      * `arabic_mmlu_high_school_european_history`
+      * `arabic_mmlu_high_school_geography`
+      * `arabic_mmlu_high_school_government_and_politics`
+      * `arabic_mmlu_high_school_macroeconomics`
+      * `arabic_mmlu_high_school_mathematics`
+      * `arabic_mmlu_high_school_microeconomics`
+      * `arabic_mmlu_high_school_physics`
+      * `arabic_mmlu_high_school_psychology`
+      * `arabic_mmlu_high_school_statistics`
+      * `arabic_mmlu_high_school_us_history`
+      * `arabic_mmlu_high_school_world_history`
+      * `arabic_mmlu_human_aging`
+      * `arabic_mmlu_human_sexuality`
+      * `arabic_mmlu_international_law`
+      * `arabic_mmlu_jurisprudence`
+      * `arabic_mmlu_logical_fallacies`
+      * `arabic_mmlu_machine_learning`
+      * `arabic_mmlu_management`
+      * `arabic_mmlu_marketing`
+      * `arabic_mmlu_medical_genetics`
+      * `arabic_mmlu_miscellaneous`
+      * `arabic_mmlu_moral_disputes`
+      * `arabic_mmlu_moral_scenarios`
+      * `arabic_mmlu_nutrition`
+      * `arabic_mmlu_philosophy`
+      * `arabic_mmlu_prehistory`
+      * `arabic_mmlu_professional_accounting`
+      * `arabic_mmlu_professional_law`
+      * `arabic_mmlu_professional_medicine`
+      * `arabic_mmlu_professional_psychology`
+      * `arabic_mmlu_public_relations`
+      * `arabic_mmlu_security_studies`
+      * `arabic_mmlu_sociology`
+      * `arabic_mmlu_us_foreign_policy`
+      * `arabic_mmlu_virology`
+      * `arabic_mmlu_world_religions`
+* `arabic_openbook_qa`: A multiple-choice openbook question answering dataset that requires external knowledge and reasoning. The open book that comes with these questions is
+  based on elementary level science facts. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
+    * Paper: https://aclanthology.org/2023.arabicnlp-1.21.pdf
+* `arabic_piqa`: Physical Interaction Question Answering (PIQA) is a multiple-choice question answering based on physical commonsense reasoning. (machine translated benchmark - part of the Alghafa Arabic translated LLM benchmark)
```

Comment:
Does this overlap with https://github.com/EleutherAI/lm-evaluation-harness/tree/main/lm_eval/tasks/alghafa ? should we deprecate those in favor of this variant?

Reply:
The dataset in https://github.com/EleutherAI/lm-evaluation-harness/tree/main/lm_eval/tasks/alghafa is more or less the same as the dataset we used, except it only contains data in test split (no data in validation, which is used for few shot evaluation). Also, we followed the process_doc to be exactly the same as what is available in lighteval (https://github.com/huggingface/lighteval). Our module contains all the benchmarks with the exact details and datasets as in the Open Arabic LLM Leaderboard and lighteval.

Reply:
got it!